exercisePPJ_06_I/ExercisePPJ_06_I.java

```diff
@@ -0,0 +1,29 @@
+/*
+Wprowadź z klawiatury wartość liczbową i zainicjuj nią zmienną typu byte o nazwie wrt.
+Następnie utwórz program, który wypisze na ekran wartości potęg dwójki lub wartości 0, dla wszystkich bitów z których składa się ta liczba.
+Przykładowo dla liczby 5 oczekiwany jest ciąg wynikowy: 0 4 0 1
+*/
+
+import java.util.Scanner;
+
+public class ExercisePPJ_06_I {
+	
+	public static void main(String[] args) {
+	
+	double binaryWrt[] = {0,0,0,0,0,0,0,0};
+	
+	System.out.print("Input of number: ");
+	Scanner input = new Scanner(System.in);
+	byte wrt = input.nextByte();
+	
+	double wrtDouble = (double) wrt;
+	
+	FunctionPPJ_06_I sendParam = new FunctionPPJ_06_I();
+	sendParam.setFunction(wrtDouble);
```

Comment:
I don't understand why you use double.
If such a size of number is not required, then int seem more reasonable data type.
Please, reply why you used double.

Reply:
input value is byte, this is a small value and integer, not float pointing, but in next situation I use Math.pow(exponent,variable) - in this moment Java see problem (convertion, variable must be a double). this is a reason why use it, and think convert variable too many times is not good, is complicated code